### ISSUE TITLE
Can't remove LG Device?

### ISSUE THREAD

**Issue by DitherMan420** [Maassoft/ColorControl#180]
How do I disable/remove the "auto detect" for LG devices? I only need Color Control for dithering. I don't want it to look for devices on my network but there seems to be no way to disable this???

**Comment by DitherMan420** [comment]
The issue is that every single time I restart my PC, if the TV is also on, I get a prompt to allow control. To be clear, is there any way to COMPLETELY DISABLE the auto detect of TVs and devices?

**Comment by DitherMan420** [comment]
The LG Controller also seems to autodettect another NON LG device on my network....which is weird. The device is a linux/android device. I really need to disable these auto detections since it's screwing up my setup, lol. I really just need this software for dithering ONLY so I'm hoping I don't have to use other software  since I really like ColorControl :-/.

**Comment by Maassoft** [comment]
Completely disabling this feature isn't possible yet.
What you could try is entering a "Device filter" on the LG controller tab which doesn't match with any of the devices on your network, then it won't detect anything. That just might stop it from doing anything with your TV or other devices.

**Comment by DitherMan420** [comment]
> 

Thanks man. I'll give it a try. Just curious, why is it picking up my Zidoo Z9X Android 9.0 streaming box? Lol. I actually got a prompt on the Zidoo similar to the LG.

**Comment by DitherMan420** [comment]
Yep the device filter did the trick!. Just curious as to why it's picking up the streaming device. Does it work with Android devices too?

**Comment by Maassoft** [comment]
No, it should not. Wasn't it a LG TV that is connected to the Zidoo?
Maybe you had the device filter empty and it found other network devices. I think I can improve the default filter by looking at the "Model" of the device (should contain "LG TV"). ColorControl simply looks at the Windows Device Manager and retrieves the devices from there.

**Comment by DitherMan420** [comment]
> No, it should not. Wasn't it a LG TV that is connected to the Zidoo? Maybe you had the device filter empty and it found other network devices. I think I can improve the default filter by looking at the "Model" of the device (should contain "LG TV"). ColorControl simply looks at the Windows Device Manager and retrieves the devices from there.

The Zidoo (linux/android) is also connected to the TV yes, however color control shows it. Is shows it as "Zidoo Z9x", with the IP address of the Zidoo. I also got a popup for access in the Zidoo. Your software is somehow also able to pick up these devices. Might be something to look into....lol.

**Comment by corckie** [comment]
> No, it should not. Wasn't it a LG TV that is connected to the Zidoo? Maybe you had the device filter empty and it found other network devices. I think I can improve the default filter by looking at the "Model" of the device (should contain "LG TV"). ColorControl simply looks at the Windows Device Manager and retrieves the devices from there.

Hey @Maassoft sorry to bother you here, but I can't seem to find any way to contact you directly and I can tell that you are active here.

I've been long time ColorControl user and would like to thank you for your work. Recently I got a little into IoT stuff and I wanted to control my TV's brightness using Node-Red. Right now I have to keep my PC open and use Color Control to adjust backlight, but using LG webos API I can't seem to get it working.

I'm still getting this error: "category, picture doesn't support the key(s): undefined".

My topic is settings/setSystemSettings and payload is {"category":"picture","settings":["brightness:30"]}

You have this sorted out. Would you please explain how to send commands to WebOS?

Again sorry for posting in this thread.

Thanks in advance.
